BnbnavNetClient/Models/CalculatedRoute.cs

```diff
@@ -438,8 +424,8 @@ void UpdateCurrentVoicePrompt()
 
         if (!Mute)
         {
-            var tts = AvaloniaLocator.Current.GetRequiredService<ITextToSpeechProvider>();
-            tts.SpeakAsync(CurrentVoicePrompt.Instruction.Speech(BlocksToNextInstruction,
+            var tts = Locator.Current.GetService<ITextToSpeechProvider>();
```

Comment:
We've been using an extension method to get classes. Should we be doing the same here?

Reply:
Since this is only done here, and the extensions fail when the services aren't present, I figured this was sufficient.